systemtest/src/main/java/io/strimzi/systemtest/utils/kafkaUtils/KafkaNodePoolUtils.java

```diff
@@ -37,4 +45,51 @@ public static void scaleKafkaNodePool(String namespaceName, String kafkaNodePool
         LOGGER.info("Scaling KafkaNodePool: {}/{} to {} replicas", namespaceName, kafkaNodePoolName, scaleToReplicas);
         KafkaNodePoolResource.kafkaNodePoolClient().inNamespace(namespaceName).withName(kafkaNodePoolName).scale(scaleToReplicas);
     }
+
+    public static void waitForKafkaNodePoolDeletion(String namespaceName, String kafkaClusterName, String kafkaNodePoolName) {
```

Comment:
I would rename it, because you are also waiting for SPS deletion.
+ I guess you can pass `podSetName` instead of `kafkaClusterName` there as a parameter